includes/functions-html.php

```diff
@@ -757,7 +760,7 @@ function yourls_html_menu() {
 	$admin_sublinks = yourls_apply_filter( 'admin_sublinks', $admin_sublinks );
 	
 	// Now output menu
-	echo '<ul id="admin_menu">'."\n";
+	echo '<nav role="navigation"><ul id="admin_menu">'."\n";
```

Comment:
> Do i convince you ?

I'm already convinced by HTML5, but just disappointed by so "huge" change on HTML4 DOM structure which can raise incompatibility issues for some old users.
I'm just saying YES for ARIA roles, but NOT SURE for some alone HTML5 elements lost in an older structure, which are two separate and different changes.